x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -2239,4 +2260,16 @@ public void testScriptsInsideAggregateFunctions_WithDateField_AndExtendedStats()
             + "InternalSqlScriptUtils.asDateTime(params.a0),InternalSqlScriptUtils.asDateTime(params.v0)))\",\"lang\":\"painless\","
             + "\"params\":{\"v0\":\"2020-05-03T00:00:00.000Z\"}},\"gap_policy\":\"skip\"}}}}}}"));
     }
+
+    public void testInOutOfRangeValues() {
```

Comment:
This test might be a good fit in `x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/PostOptimizerVerifierTests.java`?

Reply:
The logic I test here is in the `ExpressionTranslators` (`QueryTranslator` -> `ExpressionTranslators` call path) and not in the `Verifier`. Although the `PostOptimizerVerifierTests` also exercise the `QueryTranslator`, so far (and based on the name) that class only tests the `Verifier` logic. In my opinion better to keep this in the `QueryTranslatorTests`.